php-repos/phpkg
# Rethink objects

- Use the latest file-manager package
- use the latest test-runner package
- add version check to the run command
- use the existed downloaded version of the package on the run command
